diff --git a/node_modules/react-native-shared-element/RNSharedElement.podspec b/node_modules/react-native-shared-element/RNSharedElement.podspec
index b190155..aac50b0 100644
--- a/node_modules/react-native-shared-element/RNSharedElement.podspec
+++ b/node_modules/react-native-shared-element/RNSharedElement.podspec
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.platforms    = { :ios => "9.0" }
+  s.platforms    = { :ios => "9.0", :tvos => "10.0" }
 
   s.source       = { :git => "https://github.com/IjzerenHein/react-native-shared-element.git", :tag => "v#{s.version}" }
   s.source_files = "ios/**/*.{h,m}"
